osmaxx/conversion/converters/converter.py

```diff
@@ -2,7 +2,7 @@
 from osmaxx.conversion.converters import converter_gis
 from osmaxx.conversion.converters import converter_pbf
 from osmaxx.conversion.job_dispatcher.rq_dispatcher import rq_enqueue_with_settings
-from osmaxx.conversion_api.formats import FGDB, SHAPEFILE, GPKG, SPATIALITE, GARMIN, PBF
+from osmaxx.conversion.constants.formats import FGDB, SHAPEFILE, GPKG, SPATIALITE, GARMIN, PBF
```

Comment:
We could use something along the lines of `options`, since having them capitalized already makes them constants by convention. But I think it makes more sense in this case to actually use a longer name: `output_format`.